games-puzzle/amoebax/amoebax-0.2.1.recipe

```diff
@@ -0,0 +1,74 @@
+SUMMARY="A action-puzzle game, similar to tetris"
+DESCRIPTION="Amoebax is a cute and addictive action-puzzle game. \
+Due an awful mutation, some amoeba's species have started to multiply \
+until they take the world if you can't stop them. Fortunately the \
+mutation made then too unstable and lining up four or more will make \
+them disappear."
+HOMEPAGE="http://www.emma-soft.com/games/amoebax/"
+COPYRIGHT="
+	2006-2007 Jordi Fita (Code)
+	Safareig Creatiu (Graphics)
+	Àlex Almarza (Music & Sound)
+	Ferran Brugat (Web Page)
+	"
+LICENSE="
+	GNU GPL v2
+	LALv1.2
+	"
```

Comment:
Although **`PROVIDES*`**, **`REQUIRES*`**, **`BUILD_REQUIRES`** and **`BUILD_PREREQUIRES`** have newlines after the opening double quotes and before the closing double quotes, we do not use the same layout for COPYRIGHT and LICENSE. For these we use:
```
COPYRIGHT="2006-2007 Jordi Fita (Code)
	2006-2007 Safareig Creatiu (Graphics)
	2006-2007 Àlex Almarza (Music & Sound)
	2006-2007 Ferran Brugat (Web Page)"
LICENSE="GNU GPL v2
	LALv1.2"
```
or
```
COPYRIGHT="2006-2007 Jordi Fita (Code), Safareig Creatiu (Graphics), Àlex \
Almarza (Music & Sound), Ferran Brugat (Web Page)"
LICENSE="GNU GPL v2
	LALv1.2"
```
I think the second suggestion is slightly better, but I'm not sure about that.
BTW, I now a few recipes don't have the recommended layout, but for new ones it's better to stick to the recommendations ;-)